pkg/backup/item_backupper.go

```diff
@@ -458,6 +458,13 @@ func (ib *itemBackupper) takePVSnapshot(obj runtime.Unstructured, log logrus.Fie
 		return nil
 	}
 
+	// TODO: Snapshot data mover is only supported for CSI plugin scenario.
```

Comment:
This should be added after `if features.IsEnabled(velerov1api.CSIFeatureFlag)`, if the snapshot request has been handled by CSI, we will not need this warning

Reply:
Thanks. Modified.